src/btree/bt_cursor.c

```diff
@@ -1490,10 +1490,11 @@ __wt_btcur_modify(WT_CURSOR_BTREE *cbt, WT_MODIFY *entries, int nentries)
 	WT_ERR(__wt_modify_apply_api(session, cursor, entries, nentries));
 	new = cursor->value.size;
 	WT_ERR(__cursor_size_chk(session, &cursor->value));
-	if (new > orig)
-		WT_STAT_DATA_INCRV(session, cursor_update_bytes, new - orig);
-	else
-		WT_STAT_DATA_DECRV(session, cursor_update_bytes, orig - new);
+
+	WT_STAT_CONN_INCRV(session, cursor_update_bytes_changed,
+	    new > orig ? new - orig : orig - new);
+	WT_STAT_DATA_INCRV(session, cursor_update_bytes_changed,
+	    new > orig ? new - orig : orig - new);
```

Comment:
This is a funny statistic.  It now measures sum of the absolute value of all size changes.  It used to measure the actual sum of size changes.  That is, if the original size was 10 then it was 12, then 7, the net change of this statistic used to be 2 - 5 => -3.  Now it is 2 + 5 = 7.  It seems like an improvement of meaning.  If this is the intention, maybe should be called out in the ticket, as the statistics will appear to change, if anyone notices.

Reply:
ack - it was intentional. Since this was a data handle only statistic I think it's safe to change the meaning. I've called it out on the ticket as well.